src/components/header/Header.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="header">
+    <AddButton />
+    <SortButton />
+  </div>
+</template>
+
+<script>
+import AddButton from './addButton/AddButton.vue';
+import SortButton from './sortButton/SortButton.vue';
```

Comment:
bad idea to create a component for each button. You should create only 1 component